xtask/codegen/src/js_kinds_src.rs

```diff
@@ -679,6 +679,7 @@ impl Field {
                     ("||", _) => "logical_or",
                     ("&&", _) => "logical_and",
                     ("$=", _) => "suffix",
+                    ("$", LanguageKind::Graphql) => "dollar",
```

Comment:
Why was this change needed?

Reply:
Graphql use $ for parameter, and without this there will be an invalid `$_token`